tests/accuracy_tests/sim_for_alphapeel_accu_test/sim_for_accu.R

```diff
@@ -334,3 +334,644 @@ subset <- floor(seq(1, nLociAll, length.out = nSegMap))
+MF_multi_sep_genotypes <- rbind(MF_genotypes, MF_2_genotypes)
+MF_multi_sep_genotypesObs <- rbind(MF_genotypesObs, MF_2_genotypesObs)
+MF_multi_sep_genotypesObs_w_error <- rbind(MF_genotypesObs_w_error, MF_2_genotypesObs_w_error)
+MF_multi_sep_alt_allele_prob <- data.frame(matrix(nrow = 2, ncol = nLociAll))
+MF_multi_sep_alt_allele_prob[1,] <- MF_1_alt_allele_prob[, 1]
+MF_multi_sep_alt_allele_prob[2,] <- MF_2_alt_allele_prob[, 1]
+MF_multi_sep_alt_allele_prob$MF <- c("MF_1", "MF_2")
+MF_multi_sep_alt_allele_prob <- MF_multi_sep_alt_allele_prob[c(2001, 1:2000)] # Input
+MF_multi_sep_out_alt_allele_prob <- data.frame(matrix(nrow = nLociAll, ncol = 2))
+MF_multi_sep_out_alt_allele_prob[,1] <- MF_1_alt_allele_prob[, 1]
+MF_multi_sep_out_alt_allele_prob[,2] <- MF_2_alt_allele_prob[, 1] # Output
+MF_multi_sep_geno_error <- cbind(MF_geno_error, MF_2_geno_error)
+MF_multi_sep_sequenceReads <- rbind(MF_sequenceReads, MF_2_sequenceReads)
+MF_multi_sep_seq_error <- cbind(MF_seq_error, MF_2_seq_error)
+MF_multi_sep_segregation <- rbind(MF_segregation, MF_2_segregation)
+MF_multi_rec_prob <- cbind(MF_rec_prob, MF_2_rec_prob)
+
+write.table(x = MF_multi_sep_pedigree, file = "MF_multi_sep_ped_file.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_genotypesObs_w_error, file = "MF_multi_sep_geno_file.txt", 
+            row.names = TRUE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_sequenceReads, file = "MF_multi_sep_seq_file.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_alt_allele_prob, file = "MF_multi_sep_alt_allele_prob.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+
+write.table(x = MF_multi_sep_haplotypes, file = "true-MF_multi_sep_hap_0.5.txt", 
+            row.names = TRUE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_phasedGenotypes, file = "true-MF_multi_sep_phased_geno_prob.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_UnphasedGenotypes, file = "true-MF_multi_sep_geno_prob.txt",
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_genotypes, file = "true-MF_multi_sep_geno_0.3333333333333333.txt", 
+            row.names = TRUE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_genotypes, file = "true-MF_multi_sep_dosage.txt", 
+            row.names = TRUE, col.names = FALSE, quote = FALSE)
+
+write.table(x = MF_multi_sep_geno_error, file = "true-MF_multi_sep_geno_error_prob.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_seq_error, file = "true-MF_multi_sep_seq_error_prob.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
+write.table(x = MF_multi_sep_out_alt_allele_prob, file = "true-MF_multi_sep_alt_allele_prob.txt", 
+            row.names = FALSE, col.names = FALSE, quote = FALSE)
```

Comment:
I just noticed that we haven't updated our documentation for the estimated alternative allele frequency outputs. Maybe we should open an issue for this.